### PR TITLE
Set redis env variables for collection publisher

### PR DESCRIPTION
**Pull request by mgrassotti**
Set redis environment variables for collection publisher

### CODE REVIEW

File: modules/govuk/manifests/apps/collections_publisher.pp [hunk 1]
```diff
@@ -49,6 +51,14 @@
     "${title}-PUBLISHING_API_BEARER_TOKEN":
       varname => 'PUBLISHING_API_BEARER_TOKEN',
       value   => $publishing_api_bearer_token;
+    "${title}-REDIS_HOST":
+      app     => 'collections_publisher',
```
Comment (boffbowsh) [inline modules/govuk/manifests/apps/collections_publisher.pp]: Because the app name is set on line 44, it doesn't need setting here

Comment (mgrassotti) [inline modules/govuk/manifests/apps/collections_publisher.pp]: 👍

File: hieradata/common.yaml [hunk 2]
```diff
@@ -246,6 +246,9 @@ govuk::apps::bouncer::db_hostname: "transition-postgresql-slave-1.backend"
 govuk::apps::bouncer::nagios_memory_warning: 1400
 govuk::apps::bouncer::nagios_memory_critical: 1500
 
+govuk::apps::collections_publisher::redis_host: 'redis-1.backend'
```
Comment (deanwilson) [inline hieradata/common.yaml]: Should this be hard coded to a single redis when we have multiple vms running?

Comment (mgrassotti) [inline hieradata/common.yaml]: Yes, this should be hard coded as we need to use redis-1 specifically. Other apps already specify redis-1 too.

**Comment by deanwilson** [comment]
Merge when ready.

**Pull request merged**